script/build.fsx

```diff
@@ -72,11 +72,17 @@ Target "Package" (fun _ ->
     ensureDirectory artifactsNuGetDir
     ensureDirectory artifactsBuildDir
     
+    let nuspecProps = getNuspecProperties(File.ReadAllText(nuspecPath))
+    
+    let version = match TeamCityBuildNumber.IsSome with
+                      | true ->  sprintf "%s%s-apx" nuspecProps.Version TeamCityBuildNumber.Value
+                      | false -> sprintf "%s-apx" nuspecProps.Version
```

Comment:
Maybe even...

```F#
let tcBuildValue = match TeamCityBuildNumber with
                           | Some(num) -> num
                           | None -> ""

let version = sprintf "%s%s-apx" nuspecProps.Version tcBuildValue
```
I think both are equally readable, but this one compartmentalizes the functionality.